style.css

```diff
@@ -65,4 +65,328 @@
     left: 50%;
     transform: translateX(-50%);
     border-radius: 4px;
+}
+
+.header-buttons {
+    text-transform: uppercase;
+}
+
+.book-table {
+    margin-right: 40px;
+}
+
+.book-table,
+.explore {
+    font-size: 16px;
+    font-weight: 400;
+    border: 2px solid #fd5e32;
+    transition-duration: 1.1s;
+    background: transparent;
+    color: white;
+    height: 60px;
+    width: 200px;
+    margin-top: 30px;
+    border: 1px solid rgba(255, 255, 255, 0.5);
+    box-sizing: border-box;
+    border-radius: 5px;
+}
+
+.book-table:hover,
+.explore:hover {
+    background: #E8C300;
+    border-radius: 5px;
+}
+
+.header-left-text {
+    float: left;
+    font-weight: 700;
+    font-size: 12px;
+    line-height: 12px;
+    margin-left: 80px;
+    color: white;
+}
+
+.left-text {
+    transform: rotate(-90deg);
+    position: relative;
+    top: -150px;
+}
+
+.header-right-icons {
+    float: right;
+    margin-right: 80px;
+    position: relative;
+    top: -270px;
+}
```

Comment:
`header-left-text` and `header-right-icons` will be better to do with position absolute, remove floats